topostats/plottingfuncs.py

```diff
@@ -74,8 +74,8 @@ def __init__(
             The set of images to process - core or all.
         core_set : bool
             Flag to identify image as part of the core image set or not.
-        pixel_interpolation: str
-            Interpolation to use (default 'none').
+        pixel_interpolation: Union[str, None]
+            Interpolation to use (default: null).
```

Comment:
Very minor `(default: None)`.

Its a shame there is consistency between languages on what "null" is represented by.